src/behaviors/TaggableBehavior.php

```diff
@@ -52,4 +48,13 @@ public function isTagsHidden(): bool
             default => true,
         };
     }
+
+    public function isTagsSimpleHidden(): bool
+    {
+        $user = Yii::$app->user;
+        return match (str_replace('search', '', mb_strtolower($this->owner->formName()))) {
+            'clientdebt' => !$user->can('client.update'),
+            default => true,
+        };
```

Comment:
```suggestion
        if ($this->owner instanceof hipanel\modules\client\models\Client) {
            return !$user->can('client.update');
        }
```

Используй явно типы, это упрощает потом поиск мест использования и рефакторинг.
А вообще это ломает направление зависимостей. Модуль `hipanel-core` не зависит от модуля `hipanel-module-client`